client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -216,11 +216,12 @@ export default React.createClass( {
 					{ ...fieldProps( 'country-code' ) }/>
 
 				{ this.needsFax() && <Input label={ this.translate( 'Fax', { textOnly } ) } { ...fieldProps( 'fax' ) }/> }
-				<Input label={ this.translate( 'Address', { textOnly } ) } { ...fieldProps( 'address-1' ) }/>
+				<Input label={ this.translate( 'Address', { textOnly } ) } maxLength={ 40 } { ...fieldProps( 'address-1' ) }/>
```

Comment:
Hmmm, good suggestion - but this file is long as it is with various bits mixed in. Such constants would be good if it was more modular, IMHO. Maybe some day we'll refactor this file (maybe when working on Google Apps for mapped domains?), so we should keep that in mind :)